ci/azure-pipelines.yml

```diff
@@ -7,7 +7,7 @@ jobs:
 - template: azure-pipelines-template.yml
   parameters:
     name: macOS
-    vmImage: macOS-10.14
```

Comment:
Note the failure on 10.14 is occurring only on jobs macOS 1.36 and macOS nightly.

Reply:
We should wait for https://github.com/gimli-rs/object/pull/161 to land, as it fixes this problem.

Reply:
Resolved upstream